app/src/main/java/com/edu/uz/currency/currencyapp/retrofit/NbpClient.java

```diff
@@ -0,0 +1,45 @@
+package com.edu.uz.currency.currencyapp.retrofit;
```

Comment:
Pakietowałbym to bardziej po funkcjach jakie klasa pełni, czyli np. pakiety: model, service, repository, niż po nazwach bibliotek. Ale to może się @kwolkowski wypowie jak to się w Androidzie robi